src/tsserver/server.ts

```diff
@@ -967,4 +967,8 @@ namespace ts.server {
     (process as any).noAsar = true;
     // Start listening
     ioSession.listen();
+
+    if (ts.sys.tryEnableSourceMapsForHost && /^development$/i.test(ts.sys.getEnvironmentVariable("NODE_ENV"))) {
+        ts.sys.tryEnableSourceMapsForHost();
+    }

```

Comment:
While at it can you please add enabling debug info on nodes/sourcefiles/symbols as well. Thanks for doing this.
```ts
if (ts.Debug.isDebugging) {
    ts.Debug.enableDebugInfo();
}
```

Reply:
Sure